contrib/AdaEurope2018/rewrite_self_arg.adb

```diff
@@ -4,18 +4,19 @@
 with Ada.Text_IO;
 
 with Langkit_Support.Text, Libadalang.Analysis, Libadalang.Iterators,
-     Libadalang.Rewriting;
+     Libadalang.Rewriting, Libadalang.Common;
```

Comment:
Please keep the packages sorted (by name)

Reply:
Done